core/src/main/java/com/sequenceiq/cloudbreak/service/organization/OrganizationAwareResourceService.java

```diff
@@ -23,4 +25,8 @@
     T delete(T resource);
 
     T deleteByNameFromOrganization(String name, Long organizationId);
+
+    OrganizationResourceRepository<T, Long> repository();
```

Comment:
We shouldn't add the repository to the interface. We can inject all of them into the migrator.

Reply:
Thats worse than having this method in the interface. 
Also Repos don't tell what are their resources so the migrator cannot pair them to the resource type.